### ISSUE TITLE
add links to messages for matrix/irc

### ISSUE THREAD

**Issue by nirik** [fedora-infra/fmn#769]
It would be nice if there's a way to add links into the messages sent in irc/matrix. ie, 

now: 

```
FEDORA-2023-a49645772e has met stable testing requirements
```

better would be if the "FEDORA-2023-a49645772e" was a link to https://bodhi.fedoraproject.org/updates/FEDORA-2023-a49645772e

The old notifs version of that message: 

```
bodhi.update.requirements_met.stable -- fedora-release-39-0.2 reached the stable testing threshold https://bodhi.fedoraproject.org/updates/FEDORA-2023-a49645772e
```

Also, might be nice to add a prefix to messages saying what they came from? or just the topic?
ie, bodhi.update.requirements_met.stable vs ''

**Comment by ryanlerch** [comment]
@abompard should we append the schema URL property to the end of messages? most if not all the bodhi messages have this property already.

**Comment by abompard** [comment]
Yeah I thing that would be a good way to do it. :+1:

**Comment by nirik** [comment]
Yeah, in general a lot more detail on messages would be welcome. 

For example, the signing messages I just got his morning had: 

```
rpm build fedora-release-38-0.30 was signed
rpm build fedora-release-38-0.30 was signed
rpm build fedora-release-38-0.30 was signed
rpm build fedora-release-38-0.30 was signed
..
```

I think it means it signed a subpackage, but it doesn't say that. ;) 

Tagging could definitely use adding who did it: 

```
opendmarc-1.4.2-7.el9 was untagged from epel9-testing-candidate
```

(This was 'oscar' the koji garbage collector)

Mon Mar  6 07:39:32 2023 opendmarc-1.4.2-7.el9 untagged from epel9-testing-candidate by oscar

Anyhow, I can try and provide more examples if needed.

**Comment by ryanlerch** [comment]
@nirik i have filed two issues for myself in the koji messaging schemas repo to add some more details to those messages.  (as you suggested in https://github.com/fedora-infra/fmn/issues/769#issuecomment-1456578588)

**Comment by nirik** [comment]
Cool. That should help those... but not sure about others. I don't really know a good way to approach this.
I mean, I can give you a copy of the last few days of my old notifications to see how they differ... 

Not sure, but happy to help. Right now the messages are really... terse/non linked.

**Comment by ryanlerch** [comment]
> Cool. That should help those... but not sure about others. I don't really know a good way to approach this. I mean, I can give you a copy of the last few days of my old notifications to see how they differ...
> 
> Not sure, but happy to help. Right now the messages are really... terse/non linked.

yeah when i was doing the koji messages, i tried to see what they were previously from the fedmsg-meta-fedorainfrasasda repo -- but it was kind of hard to follow.

**Comment by ryanlerch** [comment]
im thinking for helping update schemas everywhere too, it might be good to include the fedora-message message id in the FMN messages (just for now) -- this will make it easier to look up the message that triggered the update in datagrepper

**Comment by nirik** [comment]
Yeah, a link to the message on datagrepper could be very handy. You could then see the entire message and figure out what info it has and perhaps why you got it and where to fix it. 

Perhaps this could be a user pref? show datagrepper link or not (defaulting to not)

**Comment by ryanlerch** [comment]
Ok, so this one is working now (mostly)

The bodhi messages now have the link at the end.

note however that the bodhi messages don't have the [Bodhi] appname prefix yet (this is implemented in the bodhi schemas, but hasnt been released yet -- anyone want to do a bodhi release)

**Comment by nirik** [comment]
nitpick: Do we need to start the app names with a capital? ie, Koji vs koji? The app is really named 'koji' not 'Koji', bodhi is really 'bodhi'. 

A bigger discussion might be if we want them to have these weird app names we have given them or use something like 'buildsys' and 'updates' ? but that might need a bigger discussion. ;( 

Finally, yeah, a new bodhi release is planned for after beta freeze. :)
